bor/client/rest/query.go

```diff
@@ -524,6 +525,14 @@ func paramsHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 	}
 }
 
+// ResponseWithHeight defines a response object type that wraps an original
+// response with a height.
+// TODO:Link it with bor
+type ResponseWithHeight struct {
```

Comment:
Do we need this code duplication?

Reply:
We need to import the other package which contain ResponseWithHeight to remove the duplication. Should I do this?

Reply:
Why not? Maybe extract this type to a new package and then import in that many places as you need to.